src/foundry/common/abstract/document.d.mts

```diff
@@ -881,7 +881,7 @@ declare abstract class Document<
 
   /**
    * @deprecated since v12, will be removed in v14
-   * @remarks `"The Document._onUpdateDocuments static method is deprecated in favor of Document._onUpdateOperation"`
+   * @remarks "The `Document._onUpdateDocuments` static method is deprecated in favor of {@link Document._onUpdateOperation | `Document._onUpdateOperation`}"
```

Comment:
Nit; you can use `@linkcode`. I suppose you may be avoiding that due to not having the TSDoc change in this PR?